driver.go

```diff
@@ -728,8 +742,24 @@ func (d *Driver) StopTask(taskID string, timeout time.Duration, signal string) e
 	if !ok {
 		return drivers.ErrTaskNotFound
 	}
+	_, err := d.podman.ContainerStats(d.ctx, handle.containerID)
```

Comment:
@towe75 what was the motivation for this check? If the container stats response is in a state we did not anticipate is there any harm in trying to stop it anyway?

Reply:
:+1: to @drewbailey's question. I'd think if the container ended up in an unexpected state before we stop it, a user would want to know.

Reply:
Oh, this change is actually part of #115 and should not have ended up here.

Reply:
👍 if you don't mind just backing it out and taking care of it in #115 that works, or we can merge #115 and rebase :)